portfolio.html

```diff
@@ -0,0 +1,114 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="normalize.css" type="text/css" rel="stylesheet">
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<nav class="navigationBar fontStyleHead">
+			<a href="#topJump" class="fontStyleHead">Home</a>
+			<a href="#aboutMeJump" class="fontStyleHead">About</a>
+			<a href="#portfolioJump" class="fontStyleHead">Portfolio</a>
+			<a href="#footerJump" class="fontStyleHead">Socials</a>
+		</div>
+		</div>
+		
+	</nav>
+	
+	<section id="topJump" class="mainImage">
+		<div class="backgroundImage">
+		<div class="overlayText">
+			<h1 class="fontStyleHead">Marcus Bowyer</h1>
+		<div class="secondOverlayText">
+			<h2 class="fontStyleHead">Full Stack Web Devloper</h2>
+		</div>
+	</div>
+	</div>
+	</div>
+	</section>
+	
+	<section id="aboutMeJump"class="aboutMe">
+		<div class="aboutMeBox fontStyleHead">
+		<h2 class="aboutMeTitle">About Me</h2>
+		<img src="https://ca.slack-edge.com/T3CSKGCFN-U019S5TT2A1-1182c80c547c-512" alt="#" class="pictureOfOwner">
```

Comment:
why is the alt tag a #?

Reply:
Was just a placeholder I forgot to change. Changed to Picture of Marcus.